<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="Dell" name="com.test.RiskReconciliationListTestcases" tests="9" failures="1" timestamp="05 Oct 2018 12:25:38 GMT" time="137.718" errors="0">
  <testcase name="NavigatePage_RiskReconciliationList" time="25.618" classname="com.test.RiskReconciliationListTestcases"/>
  <testcase name="PageHearder_RiskReconciliationList" time="5.298" classname="com.test.RiskReconciliationListTestcases">
    <failure type="java.lang.AssertionError" message="expected [Risk Reconciliation List Residual Versus Reconciled] but found [Risk Reconciliation List
Residual Versus Reconciled]">
      <![CDATA[java.lang.AssertionError: expected [Risk Reconciliation List Residual Versus Reconciled] but found [Risk Reconciliation List
Residual Versus Reconciled]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.test.WrapperClass.assertEquals(WrapperClass.java:1317)
	at com.test.RiskReconciliationListTestcases.PageHearder_RiskReconciliationList(RiskReconciliationListTestcases.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- PageHearder_RiskReconciliationList -->
  <testcase name="PanelBar_RiskReconciliationList" time="6.237" classname="com.test.RiskReconciliationListTestcases"/>
  <testcase name="PageLevelHelp_RiskReconciliationList" time="9.517" classname="com.test.RiskReconciliationListTestcases"/>
  <testcase name="RiskManagementMap_RiskReconciliationList" time="9.423" classname="com.test.RiskReconciliationListTestcases"/>
  <testcase name="QuickFilter_RiskReconciliationList" time="32.680" classname="com.test.RiskReconciliationListTestcases"/>
  <testcase name="Export_RiskReconciliationList" time="48.945" classname="com.test.RiskReconciliationListTestcases"/>
  <testcase name="Filter_RiskReconciliationList" classname="com.test.RiskReconciliationListTestcases">
    <skipped/>
  </testcase> <!-- Filter_RiskReconciliationList -->
  <testcase name="Sorting_RiskReconciliationList" classname="com.test.RiskReconciliationListTestcases">
    <skipped/>
  </testcase> <!-- Sorting_RiskReconciliationList -->
</testsuite> <!-- com.test.RiskReconciliationListTestcases -->
